docs/src/test/scala/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -416,7 +416,7 @@ class HttpServerExampleSpec extends AnyWordSpec with Matchers
         concat(
           pathEnd {
             concat(
-              (put | parameter('method ! "put")) {
+              (put | parameter('method ! "put").tmap(_ => ())) {
```

Comment:
Here the syntax change is a bit more awkward to be able to continue supporting `put | parameter(...)` which requires `Directive0` as both operands.

Reply:
Would adding a special method make it less weird?
(Either instead of `.tmap(...)` or even instead of `parameter`.)
But how would users discover what to do when it doesn't compose...

Reply:
Instead of `.tmap(_ => ()))` we could have something like `.dropExtractions` or `.dropAll`.